pkg/tsdb/influxdb/response_parser.go

```diff
@@ -114,36 +131,52 @@ func (rp *ResponseParser) buildSeriesNameFromQuery(row Row, column string) strin
 	return fmt.Sprintf("%s.%s%s", row.Name, column, tagText)
 }
 
-func (rp *ResponseParser) parseTimepoint(valuePair []interface{}, valuePosition int) (plugins.DataTimePoint, error) {
-	value := rp.parseValue(valuePair[valuePosition])
-
-	timestampNumber, ok := valuePair[0].(json.Number)
+func parseTimestamp(value interface{}) (time.Time, error) {
```

Comment:
Would be great if we could add some tests about this!

Reply:
@dsotirakis sorry, which part? i added tests for `parseTimestamp` and `parseValue`, i hope i guessed right, of not jus tell :-)